### PR TITLE
Task two finished

### PR DESCRIPTION
**Pull request by lordwerneo**

### CODE REVIEW

File: controlFlow/taskTwo/main.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+	dividedInput := strings.Split(input, " ")
+
+	for i, v := range dividedInput {
+		if i == 0 {
+			intOfValue, err := strconv.Atoi(v)
+			if err != nil {
+				log.Fatal(err)
+			}
+			min, max = int32(intOfValue), int32(intOfValue)
```
Comment (andstepko) [inline controlFlow/taskTwo/main.go:24]: It's better to leave an empty line after an error was handled, to have some logical separation and improve the readability.

File: controlFlow/taskTwo/main.go [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+	var min, max int32
+
+	dividedInput := strings.Split(input, " ")
+
+	for i, v := range dividedInput {
+		if i == 0 {
+			intOfValue, err := strconv.Atoi(v)
+			if err != nil {
+				log.Fatal(err)
+			}
```
Comment (andstepko) [inline controlFlow/taskTwo/main.go]: This block is done for each element, so can be put outside of the `if`, to avoid duplications.

**Review by andstepko — APPROVED** [review]

File: controlFlow/taskTwo/main.go (hunk 2 above)
Comment (lordwerneo) [reply controlFlow/taskTwo/main.go]: Done.

File: controlFlow/taskTwo/main.go (hunk 1 above)
Comment (lordwerneo) [reply controlFlow/taskTwo/main.go:24]: Done

**Pull request merged**